rb/spec/unit/selenium/webdriver/remote/http/default_spec.rb

```diff
@@ -31,6 +31,36 @@ module Http
             client
           end
 
+          it 'assigns default timeout to nil' do
+            http = client.send :http
+
+            expect(http.open_timeout).to eq nil
+            expect(http.read_timeout).to eq 60
+          end
+
+          describe '#initialize' do
+            let(:client) {Default.new(read_timeout: 22, open_timeout: 23)}
+            it 'accepts read timeout options' do
+              expect(client.open_timeout).to eq 23
+            end
+            it 'accepts open timeout options' do
+              expect(client.read_timeout).to eq 22
+            end
+          end
+
+          describe '#timeout=' do
+            let(:value_set) {22}
+            before {
```

Comment:
Guess this can be one-liner:

```ruby
before { client.timeout = value_set }
```

or if you want to have it multiline, please use `do ... end`:

```ruby
before do
  client.timeout = value_set
end
```